src/mono/mono.proj

```diff
@@ -252,7 +252,7 @@
       <_MonoCMakeArgs Condition="'$(Platform)' == 'arm'" Include="-DCMAKE_ANDROID_ARCH_ABI=armeabi-v7a" />
       <_MonoCMakeArgs Condition="'$(Platform)' == 'x86'" Include="-DCMAKE_ANDROID_ARCH_ABI=x86" />
       <_MonoCMakeArgs Condition="'$(Platform)' == 'x64'" Include="-DCMAKE_ANDROID_ARCH_ABI=x86_64" />
-      <_MonoCMakeArgs Include="-DENABLE_MINIMAL=ssa,portability,attach,verifier,sgen_remset,sgen_marksweep_par,sgen_marksweep_fixed,sgen_marksweep_fixed_par,sgen_copying,logging,security,shared_handles,interpreter,gac,cfgdir_config" />
+      <_MonoCMakeArgs Include="-DENABLE_MINIMAL=ssa,portability,attach,verifier,sgen_remset,sgen_marksweep_par,sgen_marksweep_fixed,sgen_marksweep_fixed_par,sgen_copying,logging,security,shared_handles,gac,cfgdir_config" />
```

Comment:
do we want the interpreter to be enabled all the time? for iOS/wasm we keep it in ENABLE_MINIMAL but set `      <_MonoCMakeArgs Include="-DENABLE_INTERP_LIB=1"/>` so that the interpreter will be built as a separate library that you can use on-demand.

Reply:
When I build it as a separate library, I hit this assertion https://github.com/dotnet/runtime/blob/095adbd3503605af2723727c5dae6d7855756a8b/src/mono/mono/mini/interp-stubs.c#L153

@vargaz are there other settings that I need?

Reply:
Turns out I wasn't properly initializing the interp.  I'm now doing

```
mono_jit_set_aot_mode(MONO_AOT_MODE_INTERP_ONLY);
mono_ee_interp_init ("");
```

And am getting an assertion:

```
D DOTNET  : assembly_preload_hook: System.ObjectModel  /data/user/0/net.dot.System.Buffers.Tests/files
D DOTNET  : assembly_preload_hook: System.Text.RegularExpressions  /data/user/0/net.dot.System.Buffers.Tests/files
D DOTNET  : ((null) error) * Assertion: should not be reached at /Users/steve/dev/net5/mobile-runtime/src/mono/mono/mini/tramp-amd64.c:1131
```

Reply:
It looks like the separate interpreter lib configuration was only tested on wasm, and it might not work elsewhere.

Reply:
Issue filed https://github.com/dotnet/runtime/issues/44804